core/modules/user/templates/user-login-form.tpl.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @file
+ * Renders the user login form.
+ *
+ * Use render($user_login_form) to print all form items, or print a subset
+ * such as render($user_login_form['name']). Always call
+ * backdrop_render_children($user_login_form) at the end in order to print all
+ * remaining items.
+ *
+ * Available variables:
+ *   - $user_login_form: An array of form items. Use render() to print them.
+ *
+ * @since 1.29.0 template added
+ * @ingroup themeable
+ */
+?>
+<?php if ($user_login_page_simplified): ?>
+  <div class="<?php print implode(' ', $classes); ?>">
+    <div class="user-login-wrapper">
+      <?php if ($logo_image): ?>
+        <div class="login-page-logo">
+          <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" class="logo">
+            <?php print $logo_image; ?>
+          </a>
+        </div>
+      <?php endif; ?>
+      <h1 class="site-name">
+        <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+      </h1>
+      <h2 class="login-page-page-title">
+        <?php print $page_title; ?>
+      </h2>
+      <div class="login-page-messages">
+        <?php print $messages; ?>
+      </div>
+<?php endif; ?>
+
+<?php print backdrop_render_children($user_login_form); ?>
```

Comment:
Am I correct in seeing this that this template only has any effect if using the "Simplified" form appearance? In which case, maybe this template should only be used in that scenario, and be named `user-simplified-page.tpl.php` (see other comment about using `#theme_wrappers`).